src/components/app/Home.js

```diff
@@ -241,15 +241,18 @@ type PopupAddonInstallPhase =
   | 'popup-enabled'
   | 'suggest-enable-popup'
   // Other browsers:
-  | 'other-browser';
+  | 'other-browser'
+  // Firefox on android:
+  | 'firefox-android';
 
 class HomeImpl extends React.PureComponent<HomeProps, HomeState> {
   constructor(props: HomeProps) {
     super(props);
     // Start by suggesting that we install the add-on.
     let popupAddonInstallPhase = 'other-browser';
-
-    if (_isFirefox()) {
+    if (_isAndroid()) {
+      popupAddonInstallPhase = 'firefox-android';
```

Comment:
I believe this is lying: we're testing only for "android", not for "Firefox on Android". Can you please move this check in the the "isFirefox" block below?

Reply:
Also it looks like the tests aren't exercizing this properly (see the codecov warning below, and the snapshot doesn't seem to contain what it should). Can you please double check this?